test/test_logging_macros.cpp

```diff
@@ -154,7 +154,7 @@ TEST_F(TestLoggingMacros, test_logging_throttle) {
     auto is_before_throttle_end =
       ((std::chrono::system_clock::now() - start) < throttle_time);
     RCUTILS_LOG_ERROR_THROTTLE(
-      RCUTILS_STEADY_TIME, throttle_time.count(), first ? "first" : "other");
+      rcutils_steady_time_now, throttle_time.count(), first ? "first" : "other");
```

Comment:
@BMarchi nit: why not keeping function behind the macro? `RCUTILS_STEADY_TIME` reads better.